### ISSUE TITLE
Rebuild server with > 1 GB RAM

### ISSUE THREAD

**Issue by tdooner** [caciviclab/disclosure-backend#67]
Our existing server has only 1 GB of RAM.

Between the InnoDB buffer pool (~130MB) and uwsgi's two worker processes (~50 MB each), this leaves very little for the extremely intensive ETL process to run. I suspect this is the cause of #66 .

@seanius You have the AWS credentials. Could you help me out by creating a new server that's at least a t2.medium? The capability to burst CPU usage seems right up our alley, since we won't need much CPU most of the rest of the time.

A t2.medium server costs ~$50 a month in us-west-1. Is this within the budget of our donated credits?

**Comment by seanius** [comment]
I suspect it's okay to do so, I'll ping the CfA persons responsible in parallel to setting it up.  Will shoot for tonight/tomorrow night.  w.r.t. credentials I shouldn't be the only one who has credentials for obvious reasons, let's sync up on slack to arrange for an N+1 holder of said credentials.

**Comment by tdooner** [comment]
Okay. Also, this is lower priority now that the import actually succeeds every night.

**Comment by seanius** [comment]
This was done, and creds handed off to tdooner.  Admin credential holders are now @tdooner , @adborden and myself

**Comment by tdooner** [comment]
Thanks @seanius !
